internal/host/validator.go

```diff
@@ -708,8 +708,9 @@ func (v *validator) isIgnitionDownloadable(c *validationContext) ValidationStatu
 	}
 	var response models.APIVipConnectivityResponse
 	if err := json.Unmarshal([]byte(c.host.APIVipConnectivity), &response); err != nil {
```

Comment:
What is actually in the field at this point?
It seems strange that this would fail to unmarshal rather than just having the validation fail.

Or even better, shouldn't we be able to detect a pending state in some better way than failing to parse the json?

Reply:
The `err` is `unexpected end of JSON input` and in the `response` object is `{ false}`

Do you think it's best not to assume that this is pending validation?

Reply:
`{ false}` looks like the default value of the response struct.
What is `c.host.APIVipConnectivity` in that case?

Reply:
Okay, maybe it makes sense then to explicitly check for empty?
That way we don't confuse an expected negative with an error (malformed json).

Reply:
Good idea! Modified, how does it look?